reark/src/main/java/io/reark/reark/data/stores/DefaultStore.java

```diff
@@ -88,7 +88,7 @@ public DefaultStore(@NonNull final StoreCoreInterface<T, U> core,
 
     @NonNull
     @Override
-    public Observable<R> getOnce(@NonNull final T id) {
+    public Flowable<R> getOnce(@NonNull final T id) {
```

Comment:
@tomaszpolanski 
I made only type changes for RxJava2.
Because I have little experience with RxJava2.